apparmor.d/groups/lxqt/lxqt-session

```diff
@@ -0,0 +1,115 @@
+
+  signal (send),
+  signal (receive) set=(kill, term) peer=startlxqt,
+  signal (receive) set=(kill, term) peer=sddm,
+
+  ptrace (read),
+
+  network netlink raw,
+
+  @{exec_path} mr,
+
+  @{sh_path}                          rix,
+  @{bin}/sed                          rix,
+  @{bin}/readlink                     rix,
+  @{bin}/dirname                      rix,
+  @{bin}/system-config-printer-applet rPx,
+  @{bin}/lxqt-config-input            rPx,
+  @{bin}/lxqt-session-settings        rPx,
+  @{bin}/lxqt-globalkeysd             rPx,
+  @{bin}/lxqt-panel                   rPx,
+  @{bin}/lxqt-policykit-agent         rPx,
+  @{bin}/lxqt-runner                  rPx,
+  @{bin}/lxqt-notificationd           rPx,
+  @{bin}/lxqt-powermanagement         rPx,
+  @{bin}/lxqt-config                  rPx,
+  @{bin}/lxqt-leave                   rPx,
+  @{bin}/lxqt-about                   rPx,
+  @{bin}/lxqt-config-monitor          rPx,
+  @{bin}/dbus-update-activation-environment rCx -> dbus,
+  @{bin}/systemctl                    rCx -> systemctl,
+
+  @{bin}/pavucontrol                  rPx,
+  @{lib}/geoclue-2.0/demos/agent      rPx,
+  @{bin}/python3.@{int}               rPx,
+  @{lib}/python3.@{int}               rPx,
+  @{bin}/nm-connection-editor         rPx,
+  @{bin}/nm-applet                    rPx,
+  @{bin}/pcmanfm-qt                   rPx,
+  @{bin}/openbox                      rix,
+  @{bin}/dconf-editor                 rPx,
+  @{bin}/setxkbmap                    rix,
+  @{bin}/start-pulseaudio-x11         rPx,
+  @{bin}/xrdb                         rPx,
+  @{bin}/xdg-user-dirs-update         rPx,
+
+  /usr/share/                         r,
+  /usr/share/mime/                    r,
+  /usr/share/cursors/                 r,
+  /usr/share/backintime/common/*      r,
+  /usr/share/desktop-directories/*    r,
+  /usr/share/system-config-printer/*  r,
+
+  /etc/xdg/                           r,
+  /etc/xdg/autostart/                 r,
+  /etc/xdg/autostart/*.desktop        r,
+  /etc/xdg/menus/lxqt-*               r,
+  /etc/xdg/openbox/*                  r,
+  /etc/udev/udev.conf                 r,
+
+  owner @{HOME}/.local/share/         r,
+  owner @{HOME}/.config/              r,
+  owner @{HOME}/.config/autostart/    r,
+  owner @{HOME}/.config/autostart/*   rw,
+  owner @{user_cache_dirs}/openbox/   rw,
+  owner @{user_cache_dirs}/openbox/sessions/          rw,
+  owner @{user_cache_dirs}/openbox/openbox.log        rwk,
+  owner @{user_config_dirs}/mimeapps.list{,.@{rand6}} rw,
+  owner @{user_config_dirs}/dconf/user r,
+  owner @{user_config_dirs}/openbox/rc.xml            r,
+  owner @{user_share_dirs}/sddm/xorg-session.log      rw,
+
+  @{PROC}/                             r,
+  @{PROC}/uptime                       r,
+  @{PROC}/@{pid}/stat                  r,
+  owner @{PROC}/@{pid}/stat            r,
+
+  @{run}/systemd/inhibit/**            rw,
+
+  /dev/tty                             rw,
+
+  include if exists <local/lxqt-session>
```

Comment:
Please more this include after the subprofile (it is where we place it for all other profiles).